README.md

```diff
@@ -49,3 +49,35 @@ make test
 
 Travis will run `make ci` to verify everything looks good.
 You can run this locally to catch errors before you push!
+
+## Deploying
+
+Once you are ready to publish `master` as a new version, run the following to bump the version in `elm-packages.json`:
+
+```
+elm-package bump
+```
+
+Commit to a branch and open a pull request. Announce the PR in #eng-front-ops-changes to have it reviewed and merged to `master`. Now you can publish the package!
+
+```
+elm-package publish # requires a tag, see below
+```
+
+This will require a tag, so follow the instructions it provides. It should be something like this:
+
+```
+Verifying NoRedInk/noredink-ui 5.10.0 ...
+Version number 5.10.0 verified (MINOR change, 5.9.1 => 5.10.0)
+Error: Libraries must be tagged in git, but tag 5.10.0 was not found.
```

Comment:
Minor comment: could put the git tag instructions before `elm-package publish` so people don't need to see this error, but I think this is fine too

Reply:
I left it in just because the error message generates the commands with the correct versions.

Reply:
Thanks for the review!